### PR TITLE
Search bar in tabs - Initial changes

### PR DESCRIPTION
**Pull request by pacproduct**
Moves the sidebar to the top and implement some safeguard so only one category can be selected at a time.

### CODE REVIEW

File: docs/settings-custom-json-all-options.md [hunk 1]
```diff
@@ -81,11 +81,11 @@ Option     | Type      | Default   | Description
 ENABLE_TREKS | Boolean | `true` | Enable treks service and fetching from the API
 ENABLE_TOURISTIC_CONTENT | Boolean | `true` | Enable touristic contents service and fetching from the API
 ENABLE_TOURISTIC_EVENTS | Boolean | `true` | Enable touristic events service and fetching from the API
-DEFAULT_ACTIVE_CATEGORIES | Array of strings (categories id) | `["T"]` | List of default active categories.
+DEFAULT_ACTIVE_CATEGORIES | Array of strings (categories id) | `["T"]` | List of default active categories.<br>**Note**: Current user interface does not make it possible to have multiple categories selected at the same time. Only the first category from this array will be taken into account as the default selected category.
```
Comment (camillemonchicourt) [inline docs/settings-custom-json-all-options.md]: Does it means that from now, this setting is used to control order of categories tabs ?

Comment (pacproduct) [reply docs/settings-custom-json-all-options.md]: One may think so, as it's an array *for legacy reasons*. But in fact, it simply allows us to indicate which category will be selected first (as in what filter will be active by default), when loading up the page.

Comment (pacproduct) [reply docs/settings-custom-json-all-options.md]: @camillemonchicourt : As it appeared not be understandable enough, we've decided to clarify this by explicitely changing the setting's name. See https://github.com/GeotrekCE/Geotrek-rando/pull/642.

Comment (camillemonchicourt) [reply docs/settings-custom-json-all-options.md]: Thanks, it looks good this way indeed.
Another question. Where can we set categories order in tabs ? 
I mean if a Geotrek-rando want to start with Treks tabs and then Products and then events and then Hostels... ?

Comment (pacproduct) [reply docs/settings-custom-json-all-options.md]: Order can be configured in Geotrek Admin.

Comment (camillemonchicourt) [reply docs/settings-custom-json-all-options.md]: Oh yes true, I didnt remembered that point about touristic content categories ordering. 
Good, but it dont allows to specify if touristic contents comes before treks, or where to order touristic events.

Comment (gutard) [reply docs/settings-custom-json-all-options.md]: Si si ! Il y a champ "ordre" sur les pratiques et sur les catégories de contenus touristique un setting TOURISTIC_EVENT_CATEGORY_ORDER pour les événements touristique. Si les numéros d'ordre des catégories de contenus touristiques sont plus petites que ceux des pratiques les CT sont bien placés avant.

Comment (camillemonchicourt) [reply docs/settings-custom-json-all-options.md]: OK merci, je ne me souvenais plus de ces aspects. Merci pour les précisions.

**Pull request merged**